source/sync/data-model/update-schema.txt

```diff
@@ -104,14 +103,13 @@ Schema Versioning Changes
 -------------------------
 
 **Schema versioning changes** are destructive changes that create a new
-compatible schema version instead of breaking your existing data model. When you deploy a schema versioning change, App Services increments the schema version and automatically handles upgrading the client-side model to the new version.
+compatible schema version instead of breaking your existing data model. When you
+deploy a schema versioning change, App Services increments the schema version
+and automatically handles upgrading the client-side model to the new version. If
```

Comment:
It would be slightly more accurate to say "When you deploy a schema versioning change, App Services increments the schema version. App Services automatically handles existing clients on older schemas in a backwards-compatible way." 

The main distinction is that the developer must manually upgrade the client to the new schema version (by making the changes in the SDK)

Reply:
Thanks for calling out. I'd definitely prefer to be as accurate as possible here, so I'll revise